spec/amp-video-interface.md

```diff
@@ -0,0 +1,66 @@
+# Video in AMP
+
+Most video player components in AMP implement the [`VideoInterface`](https://github.com/ampproject/amphtml/blob/master/src/video-interface.js) API. This means
+that a set of features is available in all of these components, either completely
+or partially.
+
+This document describes all the features (**work in progress, incomplete**) that
+these players implement.
+
+These players include:
+
+- [amp-3q-player](https://www.ampproject.org/docs/reference/components/amp-3q-player)
+- [amp-brid-player](https://www.ampproject.org/docs/reference/components/amp-brid-player)
+- [amp-dailymotion](https://www.ampproject.org/docs/reference/components/amp-dailymotion)
+- [amp-gfycat](https://www.ampproject.org/docs/reference/components/amp-gfycat)
+- [amp-ima-video](https://www.ampproject.org/docs/reference/components/amp-ima-video)
+- [amp-nexxtv-player](https://www.ampproject.org/docs/reference/components/amp-nexxtv-player)
+- [amp-ooyala-player](https://www.ampproject.org/docs/reference/components/amp-ooyala-player)
+- [amp-video](https://www.ampproject.org/docs/reference/components/amp-video)
+- [amp-wistia-player](https://www.ampproject.org/docs/reference/components/amp-wistia-player)
+- [amp-youtube](https://www.ampproject.org/docs/reference/components/amp-youtube)
+
+<a id="analytics"></a>
+
+## Analytics
+
+See [video analytics](/extensions/amp-analytics/amp-video-analytics.md).
+
+<a id="autoplay"></a>
+
+## Autoplay
+
+attribute: **`autoplay`**
+
+If this attribute is present, and the browser supports autoplay:
+
+- the video is automatically muted before autoplay starts
+- when the video is scrolled out of view, the video is paused
+- when the video is scrolled into view, the video resumes playback
+- when the user taps the video, the video is unmuted
+- if the user has interacted with the video (e.g., mutes/unmutes, pauses/resumes, etc.), and the video is scrolled in or out of view, the state of the video remains as how the user left it. For example, if the user pauses the video, then scrolls the video out of view and returns to the video, the video is still paused.
+
+For an example, visit [AMP By Example](https://ampbyexample.com/components/amp-video/#autoplay).
+
+<a id="rotate-to-fullscreen"></a>
+
+## Rotate-to-fullscreen
+
+attribute: **`rotate-to-fullscreen`**
+
+**Availability: Experimental**
+
+If this attribute is present and the video is playing manually, the video displays
```

Comment:
Please clarify what you mean by manually, that is, do you mean the user initiated the video?  If so, please rewrite as 

...and the video is playing manually (i.e., the user initiated the video)